torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3213,6 +3220,10 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCPU=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_masked_select),
+    OpInfo('matrix_exp',
+           dtypes=floating_and_complex_types_and(torch.bfloat16),
```

Comment:
From the failing tests it looks like this will need 

```
dtypesIfCPU=floating_and_complex_types_and(torch.bfloat16),
dtypesIfCUDA=floating_and_complex_types_and(torch.float16),
```

and to not set `dtypes` (just let it inherit the default)